src/css/_typography.scss

```diff
@@ -94,6 +94,26 @@ ol[class] {
     margin-top: 0;
   }
 
+  & > h1 {
+    font-size: 32px;
+    line-height: 48px;
+  }
+
+  & > h2 {
+    margin-top: 32px;
```

Comment:
Now the `h2` in markdown is styled with two rules at the same time (the other one is at line `91`). The same goes for `h3` at line `112`. Isn't that kinda confusing? 🤔 
I suggest doing all of that in "one place":
```
.markdown {
  h1 {
  // all h1 rules here
  }
  h2 {
  // all h1 rules here
  }
  h3 { ...
}
```